api/src/identity-access-management/domain/errors.js

```diff
@@ -14,6 +14,12 @@ class DifferentExternalIdentifierError extends DomainError {
   }
 }
 
+class InvalidOrAlreadyUsedEmailError extends DomainError {
+  constructor(message = 'Cette adresse e-mail est invalide ou déjà utilisée.', code = 'INVALID_OR_ALREADY_USED_EMAIL') {
+    super(message, code);
+  }
+}
```

Comment:
Il ne faut pas mettre de message en français dans les erreurs, cf. https://github.com/1024pix/pix/blob/dev/docs/adr/0044-gestion-erreurs-i18n-reference.md

Et par ailleurs il n'y a pas besoin de rendre le _« message d'erreur en anglais concis »_ et le code d'ereur configurables, aussi je suggère le code ci-dessous : 

```suggestion
class InvalidOrAlreadyUsedEmailError extends DomainError {
  constructor() {
    super('Invalid or already used e-mail address', 'INVALID_OR_ALREADY_USED_EMAIL');
  }
}
```